tests/test_account.py

```diff
@@ -843,6 +843,52 @@ def test_glue_connection_password_encryption(self):
 
         self.assertEqual(len(resources), 1)
 
+    def test_get_emr_block_public_access_configuration(self):
+        session_factory = self.replay_flight_data("test_emr_block_public_access_configuration")
+        p = self.load_policy(
+            {
+                'name': 'get-emr-block-public-access-configuration',
+                'resource': 'account',
+                'filters': [{
+                    'type': 'emr-block-public-access-configuration',
+                }]
+            },
+            session_factory=session_factory)
+        resources = p.run()
+        print(resources)
```

Comment:
minor nit, any print statements should be removed after testing

Reply:
oops good catch